File: CS/CustomDescendants/MyAdvBandedGridView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraGrid.Views.BandedGrid;
using DevExpress.XtraGrid;
using DevExpress.Utils;
using DevExpress.XtraGrid.Views.Grid.Customization;
using DevExpress.XtraGrid.Views.Base.Handler;

namespace TreeListCustomization
{
    class MyAdvBandedGridView : AdvBandedGridView
    {
        // constructors
        public MyAdvBandedGridView() : this(null) { }
        public MyAdvBandedGridView(GridControl grid) : base(grid) { }

        protected override string ViewName { get { return "MyAdvBandedGridView"; } }

        protected override CustomizationForm CreateCustomizationForm()
        {
            return new TreeListBandedCustomizationForm(this);
        }

        internal new BaseViewHandler Handler { get { return base.Handler; } }
    }
}

File: VB/CustomDescendants/MyAdvBandedGridView.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraGrid.Views.BandedGrid
Imports DevExpress.XtraGrid
Imports DevExpress.Utils
Imports DevExpress.XtraGrid.Views.Grid.Customization
Imports DevExpress.XtraGrid.Views.Base.Handler

Namespace TreeListCustomization
	Friend Class MyAdvBandedGridView
		Inherits AdvBandedGridView
		' constructors
		Public Sub New()
			Me.New(Nothing)
		End Sub
		Public Sub New(ByVal grid As GridControl)
			MyBase.New(grid)
		End Sub

		Protected Overrides ReadOnly Property ViewName() As String
			Get
				Return "MyAdvBandedGridView"
			End Get
		End Property

		Protected Overrides Function CreateCustomizationForm() As CustomizationForm
			Return New TreeListBandedCustomizationForm(Me)
		End Function

		Friend Shadows ReadOnly Property Handler() As BaseViewHandler
			Get
				Return MyBase.Handler
			End Get
		End Property
	End Class
End Namespace
